src/sentry/api/endpoints/project_repo_path_parsing.py

```diff
@@ -92,13 +91,12 @@ def repo_match(repo):
 
 class ProjectRepoPathParsingEndpoint(ProjectEndpoint):
     """
-        Returns the parameters associated with the RepositoryProjectPathConfig
-        we would create based on a particular stack trace and source code URL.
-        Does validation to make sure we have an integration and repo
-        depending on the source code URL
+    Returns the parameters associated with the RepositoryProjectPathConfig
+    we would create based on a particular stack trace and source code URL.
+    Does validation to make sure we have an integration and repo
+    depending on the source code URL
     """
 
-    @transaction_start("ProjectRepoPathParsingEndpoint")
```

Comment:
@MeredithAnya why are we removing this transaction?

Reply:
@NisanthanNanthakumar oh yeah good find I forgot to mention. When we use the `@transaction_start` with endpoints that our UI uses we are actually overriding the automatic transaction that got created which is actually not what we want. This decorator should actually only be used if we don't already automatically start a transaction. A good example was the Slack webhook endpoints.